vue.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     host: process.env.JENKINS_AGENT_NAME || 'localhost',
     proxy: process.env.NODE_ENV === 'production' ? undefined : {
       '^/api': {
-        'target': 'http://localhost:8080'
+        'target': 'https://lifelines.dev.molgenis.org'
```

Comment:
I would leave this localhost, because this backend is temporary and the default is local development in other apps.